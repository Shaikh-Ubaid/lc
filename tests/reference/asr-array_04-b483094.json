{
    "basename": "asr-array_04-b483094",
    "cmd": "lc --show-asr --no-color {infile} -o {outfile} -extra-arg=\"\"",
    "infile": "tests/../integration_tests/array_04.cpp",
    "infile_hash": "dd38786061d4a19743e4c32c2a3ecb08193777f81a2a0d3e87af0f0a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-array_04-b483094.stderr",
    "stderr_hash": "6bb018d7347e0538aafec4933b0ec3e36fa039d5498879cc9885e324",
    "returncode": -6
}