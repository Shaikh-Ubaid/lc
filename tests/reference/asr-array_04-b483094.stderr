LCOMPILERS_ASSERT failed: $DIR/src/libasr/../libasr/asr.h
function down_cast(), line number 40 at 
f != nullptr
